### PR TITLE
Per resolution at Vancouver F2F, added MAX_CLIENT_WAIT_TIMEOUT_WEBGL enu...

### PR DESCRIPTION
**Pull request by kenrussell**
...m and documentation for it.

### CODE REVIEW

File: specs/latest/2.0/index.html [hunk 1]
```diff
@@ -2425,8 +2432,10 @@ <h2 class="no-toc">Table of contents</h2>
 
     <h3><a name="CLIENT_WAIT_SYNC">clientWaitSync</a></h3>
     <p>
+
```
Comment (toji) [inline specs/latest/2.0/index.html]: Nit: Unnecessary whitespace

**Comment by toji** [comment]
LGTM. Merge at your lesiure, just in case you want to revert the whitespace nit. (I don't really care either way.)

**Comment by kenrussell** [comment]
Thanks for the review. Fixed the whitespace. Merging.
